test/test_transforms_tensor.py

```diff
@@ -608,6 +510,86 @@ def test_to_grayscale(device, Klass, meth_kwargs):
     )
 
 
+@cpu_only
```

Comment:
here as well we should parametrize with `cpu_and_gpu()` instead.

Reply:
In fact we should do it in all of the other tests here that were using `self.device`

Reply:
So the self.device had a value of 'cpu' which is why I though using cpu_only() instead of cpu_and_gpu()

Reply:
Yes, but as I mentioned above all these tests were called twice: once as part of `Tester` with 'cpu', and once as part of `CudaTester` with 'cuda'. Which is why we need to parametrize over 'device' with `cpu_and_gpu()` now :)

Reply:
Okay, thanks for clarifying :)